PathfinderHonorManager/Startup.cs

```diff
@@ -47,8 +50,7 @@ public void ConfigureServices(IServiceCollection services)
                 .AddScoped<IHonorService, HonorService>()
                 .AddScoped<IPathfinderHonorService, PathfinderHonorService>();
             services.AddMvc()
-                .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<PathfinderValidator>())
-                .AddFluentValidation(fv => fv.RegisterValidatorsFromAssemblyContaining<PathfinderHonorValidator>());
```

Comment:
It looks like FluentValidation scans the entire assembly when `RegisterValidatorsFromAssemblyContaining` is called. De-dupped this.